orchestrator/src/indexer/rooch.ts

```diff
@@ -238,32 +238,58 @@ export default class RoochIndexer extends Indexer {
       return null;
     }
 
+    const keeper_key = await prismaClient.keeper.upsert({
+      where: {
+        chain_module: {
+          module: data.notify ?? "",
+          chain: this.chainId,
+        },
+      },
+      create: {
+        module: data.notify ?? "",
+        chain: this.chainId,
+        privateKey: Secp256k1Keypair.generate().getSecretKey(),
```

Comment:
Does this mean that a new keypair is created for every single notification via the Keeper?

Should it not be a single keypair per destination notified Smart Contract?

Reply:
Its not every single time, Upsert is update or Insert
and the update on L256 is empty

Reply:
@rsoury

Reply:
Got it.